cfme/infrastructure/datastore.py

```diff
@@ -118,7 +118,7 @@ class DatastoreCollection(Navigatable):
     """Collection class for `cfme.infrastructure.datastore.Datastore`"""
 
     def __init__(self, appliance=None):
-        self.appliance = appliance
+        Navigatable.__init__(self, appliance=appliance)
```

Comment:
`__init__` can be removed.

Reply:
agree but I would leave it for readability.